ee/localserver/krypto-ec-middleware.go

```diff
@@ -459,3 +477,197 @@ func (bhr *bufferedHttpResponse) WriteHeader(code int) {
 func (bhr *bufferedHttpResponse) Bytes() []byte {
 	return bhr.buf.Bytes()
 }
+
+func (e *kryptoEcMiddleware) presenceDetectionCallback(challengeBox *challenge.OuterChallenge) {
+	// extract cmd req from challenge box
+	ctx, cancel := context.WithTimeout(context.Background(), presencedetection.DetectionTimeout)
+	defer cancel()
+
+	var cmdReq v2CmdRequestType
+	if err := json.Unmarshal(challengeBox.RequestData(), &cmdReq); err != nil {
+		e.slogger.ErrorContext(ctx,
```

Comment:
I think we've been standardizing on `Log(ctx, slog.LevelError, ...)` over e.g. `ErrorContext`. Same comment for other calls to `ErrorContext`, `DebugContext`, `InfoContext` below.

Reply:
yep, I think I have said those exact words before =) fixed